src/Goerli-DssSpell.sol

```diff
@@ -36,6 +36,23 @@ contract DssSpellAction is DssAction, DssSpellCollateralOnboardingAction {
     // Provides a descriptive tag for bot consumption
     string public constant override description = "Goerli Spell";
 
+    // Math
+    uint256 constant RAY  = 10 ** 27;
+
+    // --- Offboarding: Current Liquidation Ratio ---
+    uint256 constant CURRENT_UNI_A_MAT              =  165 * RAY / 100;
```

Comment:
as per the spell sheet

```suggestion
    uint256 constant CURRENT_UNI_A_MAT              =  145 * RAY / 100;
```

Reply:
Are you sure about this one? These values are used for `UNIV2WBTCETH-A`. Also i check `spotter` and it has 165 for UNI-A

Reply:
Looks like there's a copy paste error in the implementation sheet. Correct values are found here as per the SR:
https://docs.google.com/spreadsheets/d/1miE1AOJsOXDXbcIDGvJkAefRChC7q_X713GvX00k_pM/edit#gid=1101084058